pkg/schema/schema.go

```diff
@@ -2,36 +2,38 @@
 
```

Comment:
didn't you have to change the json file?

Reply:
Yes, I have to change